deepchem/feat/fingerprints.py

```diff
@@ -37,17 +43,12 @@ class CircularFingerprint(Featurizer):
   """
   name = 'circular'
 
-  def __init__(self,
-               radius=2,
-               size=2048,
-               chiral=False,
-               bonds=True,
-               features=False,
-               sparse=False,
-               smiles=False):
+  def __init__(self, radius=2, size=2048, chiral=False, counts=False, bonds=True,
+             features=False, sparse=False, smiles=False):
```

Comment:
When adding a new argument, it's best to put it at the end of the list.  Otherwise, you'll break existing code that was calling the function and not specifying the arguments by name.

Reply:
Thanks, I wasn't aware of that. I fixed it